src/actions/breakpoints/addBreakpoint.js

```diff
@@ -52,6 +53,21 @@ export default async function addBreakpoint(
   const symbols = getSymbols(getState(), source);
   const astLocation = await getASTLocation(source, symbols, newLocation);
 
+  // generatedSource.text, in the case of pretty print or original tab
+  // returns the minified contents, which is why it's used
+  // when creating the "originalText"
```

Comment:
Why don't we get the original text from the original source? `source`

Reply:
As I mentioned last week, `addBreakpoint` is receiving `:formatted` (pretty-printed) sources.  I wish that wasn't the case, so this task would be as easy just using `originalSource` and `generatedSource`, but this has been part of what's delaying this problem.